Improve wallets documentation

martonlederer opened the pull request:
- Modify links to point to the main wallets FAQ homepage
- Add separate guide links to Arweave.app ~~and ArConnect~~
- Move Arweave.app FAQ questions to it's guide page
~~- Add guide to ArConnect~~

rosmcmahon commented on wallets/arweave-wallet.md:
```diff
@@ -84,4 +84,10 @@ To remove a set of wallet keys from the wallet app, click the <img src="../.gitb
 
 From this screen you'll be able to re-download the key file for a particular wallet key pair or click the "Delete" button to remove it from the wallet app entirely. As long as you have your passphrase or key file you'll be able to add it back whenever you like so keep those safe and secure.
 
+## **Using Arweave.app as a cold wallet**
```
this needs to stay in FAQs. it's the most popular question, and has even got it's own page.

rosmcmahon commented on wallets/wallet-faq.md:
```diff
@@ -8,17 +8,66 @@ description: All questions surrounding wallets
-Yes, you can find detailed instructions for how to securely generate an Arweave cold wallet with the arweave.app [**here**](https://docs.arweave.org/info/wallets/generating-cold-wallet).
-
-**\* Please make sure to keep multiple copies of your wallet key file in secure locations, as we cannot help you to recover your wallet if you lose it \***
+<table>
```
these FAQs need to stay.

rosmcmahon commented on wallets/wallet-faq.md:
```diff
@@ -8,17 +8,66 @@ description: All questions surrounding wallets
-
-Yes, you can find detailed instructions for how to securely generate an Arweave cold wallet with the arweave.app [**here**](https://docs.arweave.org/info/wallets/generating-cold-wallet).
-
-**\* Please make sure to keep multiple copies of your wallet key file in secure locations, as we cannot help you to recover your wallet if you lose it \***
```
what is the benefit to users of removing this?

rosmcmahon commented:
DMed you

martonlederer commented:
Fixed up issues:
- Added back content to wallet FAQ (+improvements and extra FAQ)
- Added guides for generating cold wallets with ArConnect and the ArDrive CLI (the ArDrive CLI is considered advanced, so I added a note about this in the beginning of the guide)
- Added images for the cold wallet generation guide (both for Arweave.app and ArConnect)
- Finished up the ArConnect guide (setup section, disclaimer about stored wallets, manage wallets section, settings section)

martonlederer commented:
- Moved ArConnect docs to #20

rosmcmahon commented:
as discussed this PR will not be merged.
